reference/compiling_for_android.rst

```diff
@@ -215,14 +198,14 @@ failure should be presented there.
 
 Seek assistance if you can't figure it out.
 
-Application exits immediately
+2.) Application exits immediately
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 If the application runs but exits immediately, there might be one of the
 following reasons:
 
--  libgodot_android.so is not in ``libs/armeabi``
+-  libgodot_android.so is not in ``lib/armeabi-v7a``
 -  Device does not support armv7 (try compiling yourself for armv6)
--  Device is Intel, and apk is compiled for ARM.
+-  Device is ARM, and apk is compiled for intel.
```

Comment:
That's indeed biased. Most devices are ARM, and the default build options are for ARM, so in the troubleshooting section, one should mention Intel as it was.